src/SonarScanner.MSBuild.Shim/PropertiesFileGenerator.cs

```diff
@@ -181,16 +181,21 @@ private ICollection<FileInfo> PutFilesToRightModuleOrRoot(IEnumerable<ProjectDat
                 if (!file.Exists)
                 {
                     logger.LogWarning(Resources.WARN_FileDoesNotExist, file);
-                    logger.LogDebug(Resources.DEBUG_FileReferencedByProjects, string.Join("', '",
-                        group.Value.Select(x => x.Project.FullPath)));
+                    logger.LogDebug(
+                        Resources.DEBUG_FileReferencedByProjects,
+                        string.Join("', '", group.Value.Select(x => x.Project.FullPath)));
                     continue;
                 }
```

Comment:
based on [this comment](https://github.com/SonarSource/sonar-scanner-msbuild/issues/1032#issuecomment-1111078148), we can directly exclude files ending with `DLL` and `EXE`

Reply:
Some warnings mentioned in #1032 are like:

> 13:40:17.161  13:40:17.161  WARNING: File 'C:\Users\user\.nuget\packages\microsoft.testplatform.testhost\16.10.0\build\netcoreapp2.1\x64\Microsoft.TestPlatform.PlatformAbstractions.dll' is not located under the root directory 'C:\src\POC1\POC1' and will not be analyzed.

We cannot analyze DLL files or EXE, so this is pure noise.

It doesn't matter if they are real, if they are outside of the root or inside the root. We cannot analyze DDLs and EXE files.